src/com/itacademy/eshop/product/Product.java

```diff
@@ -48,7 +54,15 @@ public Integer getAverageRating() {
         return rating / reviews.size();
     }
 
-    public void setPrice(int price) {
+    public void setPrice(double price) {
         this.price = price;
     }
+
+    public double getTotalPrice() {
+        double totalPrice = price;
+        if (availability == Availability.IN_STOCK) {
```

Comment:
Since availability can be null. I suggest to have nullsafe equals like this `Availability.IN_STOCK.equals(availability)`
This way even if availability is null you will get false. Otherwise it would result in nullpointer exception